os/include/sys/socket.h

```diff
@@ -254,6 +293,9 @@ int getsockname(int sockfd, FAR struct sockaddr *addr, FAR socklen_t *addrlen);
 /**
 * @brief  get the name of the peer socket
 *
+* @details @b #include <sys/socket.h>\n
+* SYSTEM CALL API\n
```

Comment:
please check if this API is SYSTEM CALL or not.

Reply:
revised